CHANGELOG.md

```diff
@@ -30,7 +30,7 @@ bumped version for float/double range bugfix
 2. Support for monadic generators e.g. [mwc-random](https://hackage.haskell.org/package/mwc-random).
 3. Monadic adapters for pure generators (providing a uniform monadic
    interface to pure and monadic generators).
-4. Faster by more x10 (depending on the type) - see below for benchmarks.
+4. Faster by more x1000 (depending on the type) - see below for benchmarks.
```

Comment:
Super-nit: I tend to be skeptical of claims like "better by up to <number>". In this case, even the _least_ improved generator is impressive, so I think we should mention it. i.e. "40 to 1000 times faster pseudo-random value generation". (I think 40 is about right, but worth checking again).

Reply:
Well the minimum is INTEGER which is only 13% faster - the next lowest is x18 for types with 8 bits - I think if this is too heavily caveated you lose the message

Reply:
Fair enough. No need to dwell on this too much. The other alternative that comes to mind would be e.g. "Faster generation for all implemented types, by up to x1000 for fixed-width integral types".

But if you think that's too much caveating, I'm fine with keeping this as it is.

Reply:
Maybe something vague, but still accurate with a lower bound, along the lines: "Performance improvements all across the board with generation of most types being faster on the orders of x100 to x1000"